thirdeye-ui/src/app/pages/alerts-all-page/alerts-all-page.component.tsx

```diff
@@ -192,6 +212,22 @@ export const AlertsAllPage: FunctionComponent = () => {
         });
     };
 
+    // Since the alert accuracy is fetched via a different API, that data is not being injected
+    // into the main uiAnomaly state to avoid adding complexity to the state updated; instead,
+    // the accuracyStatistics data is stored in alertsAccuracy and injected into the uiAlerts state
+    //  data asynchronously, as and when it is resolved before that is passed on to the to the
+    // AlertListV1 component. This separate storage of accuracy data in this component might
+    // not be especially advantageous right now, but it will be helpful if the alerts API needs
+    // to be called multiple times, say for searching and filtering
+    const uiAlertsWithAccuracy = useMemo<typeof uiAlerts>(
+        () =>
+            uiAlerts?.map((uiAlert) => ({
+                ...uiAlert,
+                accuracyStatistics: alertsStats[uiAlert.id],
+            })) ?? null,
+        [uiAlerts, alertsStats]
+    );
```

Comment:
This is also required since `uiAlerts` can be modified when users delete an alert